less/index.less

```diff
@@ -56,4 +53,168 @@ table {
 html, body {
     height: 100%;
 }
+html{
+	font-size:62.5%;
+}
+p{
+	font-size:@baseFontSize;
+}
+li{
+	font-size:@baseFontSize;
+}
+// Mixin declarations
+.center(){
+	display:flex;
+	justify-content: center;
+	align-items:center;
+}
+.aTagStyle{
+	font-size:@baseFontSize;
+	color:black;
+	text-decoration:none;
+}
+// these were mostly just for practice
+@baseFontSize:1.6rem;
+@h2FontSize:2.5rem;
+@bgColor:lightblue;
+
+// Home Page CSS
+// gotta love these single line comments!
+.container{
+	max-width:880px;
+	margin:0 auto;
+	@media(max-width:500px){
+		max-width:500px;
```

Comment:
you don't need to do this. this media query is only effective up to 500px, so setting the max-width to 500px doesn't actually do anything